docs/poc2023q1/example1.md

```diff
@@ -0,0 +1,256 @@
+   EdgePlacement for the green workload.  TODO later: add an object
+   that prescribes customization.  TODO later: include an object that
+   prescribes summarization.
+   
+    ```shell
+    kubectl ws root
+    kubectl ws create work-g --enter
+    kubectl create ns greenstuff
+    kubectl label ns greenstuff common=no
+    kubectl create deployment --image=gcr.io/kuar-demo/kuard-amd64:green --port=8080 greend # TODO: make it in the greenstuff namespace
+    kubectl create -f - <<<EOF
+    apiVersion: edge.kcp.io/v1alpha1
+    kind: EdgePlacement
+    metadata:
+      name: edge-placement-g
+    spec:
+      locationSelectors:
+        matchLabels: {"env":"prod","extended":"yes"}
+      namespaceSelector: {"common":"no"}
+    EOF
+    ```
+
+3. In response to each EdgePlacement, the edge scheduler will create a
+   corresponding SinglePlacementSlice object.  These will indicate the
+   following resolutions of the "where" predicates.
+
+   | EdgePlacement | Resolved Where |
+   | ------------- | -------------- |
+   | edge-placement-b | kind1, kind3 |
+   | edge-placement-g | kind3 |
+   
+   Check out the SinglePlacementSlice objects as follows.
+   
+    ```shell
+    $ kubectl get SinglePlacementSlice
+    # TODO: show what they look like
+    ```
+
+## Stage 3
+
+![Placement translation](Edge-PoC-2023q1-Scenario-1-stage-3.svg "Stage
+3 summary")
+
+Stage 3 has the following steps.
+
+In response to the EdgePlacement and SinglePlacementSlice objects, the
+placement translator will copy the workloads into the mailbox
+workspaces and create TMC placement objects there.
+
+The blue workload goes only to the kind1 cluster.  Examine its TMC
+Placement object and blue workload as follows.
+
+```shell
+$ kubectl ws root:edge:niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368
+$ kubectl get Placement -o yaml
+TODO: show what it looks like
+$ kubectl get ns
+(will list the syncer workspace and bluestuff)
+$ kubectl get Deployment -A
+(will list the syncer and blued Deployments)
+```
+
+The kind3 cluster gets both the blue and gree workloads.  Examine its
+TMC Placement object and workloads as follows.
+
+```shell
+$ kubectl ws root:edge:niqdko2g2pwoadfb-mb-c5820696-016b-41f6-b676-d7c0ef02fc5a
+$ kubectl get Placement -o yaml
+TODO: show what it looks like
+$ kubectl get ns
+(will list the syncer workspace, bluestuff, and greenstuff)
+$ kubectl get Deployment -A
+(will list the syncer, blued, and greend Deployments)
+```
+
+## Stage 4
+
+![TMC kicks in](Edge-PoC-2023q1-Scenario-1-stage-4.svg "Stage 4 summary")
+
+Stage 4 has the following steps.
+
+TMC does its thing between the kind1 cluster and its mailbox
+workspace.  This is driven by the one TMC Placement object in that
+mailbox workspace.
+
+TMC does its thing between the kind3 cluster and its mailbox
+workspace.  This is driven by the two TMC Placement objects in that
+mailbox workspace, one for the blue workload and one for the green
+workload.
+
+## Stage 5
```

Comment:
The arrows in the diagram for 5b show status for MBG going back to WMWSB.  Why?  Shouldn't those status go back to WMWSG?